### DISCUSSION TITLE
Alwrity's AI Blog Writer is Generating Super Wow Human  Like Content: Passed 100% AI Detection and 100% Plagiarism Test

### DISCUSSION THREAD

**Discussion by uniqueumesh** [ALwrity/ALwrity#60] [General]
I gave this Title to Alwrity's AI blog writer: "How Can AI Tools Benefit Writers?" and it generated a human-like blog. I am really happy with the content. After generating, I tested this blog on the AI detection tool and Plagiarism tool. This content passed with 100% human-written and 100% Unique. I have given the screenshot of this test below. Revolution is here 🎉

![Screenshot (105)](https://github.com/AJaySi/AI-Writer/assets/150319020/3a9538ed-094e-49e2-be9f-3f567e836fc1)

![Screenshot (106)](https://github.com/AJaySi/AI-Writer/assets/150319020/2453bbed-06f4-464c-980b-95f74ebbc137)

![Screenshot (108)](https://github.com/AJaySi/AI-Writer/assets/150319020/ef943530-e9c9-4923-afb1-1bcce9b00ba2)

**Comment by AJaySi** [comment]
@umesh070  
Thanks a lot. And thanks for being supportive AI engineer & Quality Guardian.
There is so much more improvements to do. Alwrity can write a lot better.

Note: Content should be good, albeit written by AI. To err is Human & AI...

**Comment by Strickland808** [comment]
This is a great ai writting tool with lots of useful features. Unfortunately it still doesn't produce human like content, I ran some tests and everything it produced came back 100% ai content on quillbot, copyscape and scribler.

**Comment by AJaySi** [comment]
@Strickland808 : Hello, hope you are doing fine.

Options to improve the quality.
Option1:
a). Visit alwrity.com and checkout the online blog writer. Above results are from there.
b). Use more powerful AI models like chatgpt4o & Gemini Pro flash

Option2: Commandline Options: Checkout main_config file and make changes. 
Try Agents team & Long form writers.
 
Ultimately, better prompts will give better content. Let us know, We will guide you through prompting and you can help make alwrity better for everyone. 

Note: Alwrity is a writing research assistant, presently focusing on well researched  & high quality content.
But, we will have to work on 'human-like' content & personlization. Hoping, AI Agents team will solve that problem for us.
Checkout this thread too: https://github.com/AJaySi/AI-Writer/discussions/52#discussioncomment-9498310

Regards.
